internal/cluster/cluster.go

```diff
@@ -25,9 +25,9 @@ import (
 
 func IsValidID(clusterID string) error {
 	// Make sure the clusterid is a valid DNS-1123 string
-	if match, _ := regexp.MatchString("^[a-z0-9][a-z0-9.-]*[a-z0-9]$", clusterID); !match {
+	if match, _ := regexp.MatchString("^[a-z0-9][a-z0-9-]*[a-z0-9]$", clusterID); !match {
```

Comment:
Is suggest adding a limit on the length as well, as we discussed https://github.com/submariner-io/lighthouse/issues/707#issuecomment-1068888749